.github/workflows/publish.yml

```diff
@@ -8,6 +8,62 @@ on:
 jobs:
   tests:
     uses: ./.github/workflows/test.yml
+  
+  publish-installer-release:
+    needs: tests
+    strategy:
+      fail-fast: false
+      matrix:
+        job:
+          - { target: x86_64-unknown-linux, exe: amd64-linux, os: ubuntu-latest }
+          - { target: aarch64-unknown-linux, exe: aarch64-linux, os: ubuntu-latest }
+          - { target: x86_64-apple-darwin, exe: macos, os: macos-latest }
+          - { target: x86_64-pc-windows-msvc, exe: windows.exe, os: windows-latest }
+    runs-on: ${{ matrix.job.os }}
+    defaults:
+      run:
+        working-directory: ./micro-rdk-installer
+    steps:
+    - name: Checkout main branch code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
+    - name: Set toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: 1.62.0
+        override: true
+        target: ${{ matrix.job.target }}
+    - name: Build installer binary
+      uses: actions-rs/cargo@v1
+      with:
+        use-cross: true
+        args: --release --target=${{ matrix.job.target }}
+        command: build 
+    - name: Rename result
```

Comment:
i think this is going to clash (remains to try) with the release of the micro-rdk binary so we want to make sure to test that otherwise integrate both workflow together